neo/SmartContract/Native/Tokens/Nep5Token.cs

```diff
@@ -73,6 +73,32 @@ internal protected virtual void MintTokens(ApplicationEngine engine, UInt160 acc
             engine.SendNotification(ScriptHash, new StackItem[] { "Transfer", StackItem.Null, account.ToArray(), amount });
         }
 
+        internal protected virtual void Burn(ApplicationEngine engine, UInt160 account, BigInteger amount)
+        {
+            if (amount.Sign < 0) throw new ArgumentOutOfRangeException(nameof(amount));
+            if (amount.IsZero) return;
+            StorageKey key = CreateAccountKey(account);
+            StorageItem storage = engine.Snapshot.Storages.GetAndChange(key);
```

Comment:
without factory, this will throw an exception, i think that this is expected

Reply:
Yes, this is expected.